src/Driver/Callbacks/Callbacks.jl

```diff
@@ -85,7 +86,29 @@ function GenericCallbacks.call!(cb::SummaryLogCallback, solver, Q, param, t)
         estimated_remaining,
         normQ,
     )
-    isnan(normQ) && error("norm(Q) is NaN")
+
+    if isnan(normQ)
+        vs = vars(Q)
+        nan_fields = []
+        for ftc in flattened_tup_chain(vs)
+            i_vars = varsindex(vs, wrap_val.(ftc)...)
+            if length(i_vars) > 1 # array
+                for (j, i_var) in enumerate(i_vars)
+                    if isnan(norm(Q[:, :, i_var]))
```

Comment:
```suggestion
                    if isnan(norm(Array(Q[:, :, i_var])))
```
Better to make this GPU friendly